workflow-event-listener/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWithTTLWorkflowStatusListener.java

```diff
@@ -78,7 +78,7 @@ public void shutdownExecutorService() {
 
     @Override
     public void onWorkflowCompleted(WorkflowModel workflow) {
-        LOGGER.info("Archiving workflow {} on completion ", workflow.getWorkflowId());
```

Comment:
I would tweak this a little bit.

LOGGER.info("Archiving workflow. Workflow Name : {} Workflow Id : {} Workflow Status : {}", workflow.getWorkflowName(), workflow.getWorkflowId(), <workflow status>);

Reply:
done